Both interegular and Js2Py are not needed to run tests.

--- lark-1.3.1/test-requirements.txt.orig
+++ lark-1.3.1/test-requirements.txt
@@ -1,3 +1 @@
-interegular>=0.3.1,<0.4.0
-Js2Py==0.68
 regex
